### DISCUSSION TITLE
Possibility to associate a "Name" (Payee) to 2 "Assett accounts" type

### DISCUSSION THREAD

**Discussion by MAPA2023** [moneymanagerex/moneymanagerex#5935] [Polls]
Hi everyone,

I submit this idea that allows you to get even more flexibility in this beautiful project.

As you know in mmex, an accounting entry of "Deposit" or "Withdrawal" is always based on various factors among which there are mandatory: "Account" (Asset - bank etc..), Name (Payee or From), "Category " (Revenue or Cost).

Well, for example, I need to use an "Asset Account" where I want to convey the money received, for example, from a bank associated with a specific name (From) and then have the costs exit from that account. This allows me to monitor the account for that particular name!

Currently this is not possible with a single transaction that moves "two accounts" of the patrimonial type associated with a "Name" (From or Payee). 

Two obvious optimization implementations are possible at the discretion of the engineers:
1a in the "Transfer" procedure, add the "Name" (Payee or From) like to "Category" but it is optional and not mandatory.

2s in the "Deposit" or "Withdrawal" procedure, add in the "Category" the possibility of viewing the balance sheet accounts, for example set to be viewed.

I am attaching hypothetical vision photos.

What do you think?

EX OPTION A

![Transerf A](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/1ccf6c10-03ff-4f78-aca6-9f52a16df443)

EX OPTION B

![Transaction 2](https://github.com/moneymanagerex/moneymanagerex/assets/128650264/cf10fc13-8edc-428a-8a68-374a0cf8dfb3)

**Comment by tactilis** [comment]
Note that this Poll relates to @MAPA2023's Discussion in https://github.com/moneymanagerex/moneymanagerex/discussions/5912
